default/python/universe_generation/empires.py

```diff
@@ -422,10 +422,9 @@ def setup_empire(empire, empire_name, home_system, starting_species, player_name
         print "Player", player_name, ": no available foci on homeworld for starting species", starting_species
 
     # give homeworld starting buildings
-    # use the list provided in scripting/starting_unlocks/buildings.inf
     print "Player", player_name, ": add starting buildings to homeworld"
-    for building in load_string_list(os.path.join(fo.get_resource_dir(), "scripting/starting_unlocks/buildings.inf")):
-        fo.create_building(building, homeworld, empire)
+    for item in fo.load_starting_buildings():
+        fo.create_building(item.name, homeworld, empire)
```

Comment:
Which is why using the unlocked items parser for this is not ideal. The starting buildings file has always been a simple list of building names, which avoided issues like that.